autoload/neomake/makers/ft/zsh.vim

```diff
@@ -1,19 +1,9 @@
 " vim: ts=4 sw=4 et
 
-function! neomake#makers#ft#zsh#SupersetOf() abort
-    return 'sh'
-endfunction
-
 function! neomake#makers#ft#zsh#EnabledMakers() abort
     return ['zsh']
 endfunction
 
-function! neomake#makers#ft#zsh#shellcheck() abort
-    let maker = neomake#makers#ft#sh#shellcheck()
-    let maker.args += ['--shell', 'zsh']
-    return maker
-endfunction
```

Comment:
I would keep this.
https://github.com/neomake/neomake/commit/fbc0b9fb001071afbbba1a6aebcc0c0f49338697 was meant to allow using it, but I've not intended to have it enabled by default.

So I suggest to just remov the `SupersetOf` for now.

Reply:
sounds good